index.html

```diff
@@ -20,7 +20,7 @@
         <ul class="nav__list">
           <li class="nav__list--item"><a href="#home">Home</a></li>
           <li class="nav__list--item"><a href="#features">Features</a></li>
-          <li class="nav__list--item"><a href="#description">Description</a></li>
+          <li class="nav__list--item"><a href="#funcionality">Description</a></li>
```

Comment:
Zmienilam na funcionality, zeby bylo in line z semantyka jaka uzywa Lukasz w swojej czesci